src/pages/data-management/index.jsx

```diff
@@ -19,53 +18,34 @@ import ExampleExperimentsSpace from 'components/data-management/ExampleExperimen
 
 const DataManagementPage = () => {
   const dispatch = useDispatch();
-  const projectsList = useSelector(((state) => state.projects));
 
-  const {
-    saving: projectSaving,
-  } = projectsList.meta;
+  const samples = useSelector((state) => state.samples);
 
-  const sampleSaving = useSelector((state) => state.samples.meta.saving);
+  const { activeExperimentId } = useSelector((state) => state.experiments.meta);
+  const experiments = useSelector(((state) => state.experiments));
 
-  const { activeProjectUuid } = useSelector((state) => state.projects.meta);
+  const activeExperiment = experiments[activeExperimentId];
+  const { saving: experimentsSaving } = experiments.meta;
+  const { saving: samplesSaving } = samples.meta;
 
-  const experiments = useSelector((state) => state.experiments);
   const [newProjectModalVisible, setNewProjectModalVisible] = useState(false);
-  const activeProject = projectsList[activeProjectUuid];
-  const loadedSamples = useSelector((state) => state.samples);
-
-  const experimentIds = new Set(experiments.ids);
-  const experimentsAreLoaded = activeProject?.experiments
-    .every((experimentId) => experimentIds.has(experimentId));
 
   useEffect(() => {
-    if (projectsList.ids.length === 0) dispatch(loadProjects());
+    if (experiments.ids.length === 0) dispatch(loadExperiments());
   }, []);
 
-  const updateRunStatus = (experimentId) => {
-    dispatch(loadBackendStatus(experimentId));
-  };
-
-  const samplesAreLoaded = () => !activeProject.samples.length
-    || activeProject.samples.every((sample) => Object.keys(loadedSamples).includes(sample));
+  const sampleAreLoaded = () => !activeExperiment.sampleIds.length
```

Comment:
Can you rewrite this function so that it's easier to understand? The first clause of the or clause means that if there are no samples, samples should be say to be loaded. However, I think the intention is not very clear.

Reply:
I don't think we really need the first of the or clauses honestly, if there are no `sampleIds` the `every` will just return true and that's it, so we can remove the first part of the clause